normandy/base/api/authentication.py

```diff
@@ -92,8 +98,12 @@ def fetch_oidc_user_profile(self, access_token):
         response = requests.get(url, headers={"Authorization": f"Bearer {access_token}"})
 
         if response.status_code == 200:
+            now = int(time.mktime(datetime.utcnow().timetuple()))
+            ratelimit_expires_in = int(response.headers.get("X-RateLimit-Reset", 0)) - now
```

Comment:
A default of 0 is probably going to do weird things, like set the cache to expire in negative 50 years. Maybe the default should be now Or now + 60?